msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/ResetPasswordStates.kt

```diff
@@ -361,4 +391,22 @@ class ResetPasswordPasswordRequiredState internal constructor(
             }
         }
     }
+
+    override fun writeToParcel(parcel: Parcel, flags: Int) {
+        parcel.writeString(flowToken)
```

Comment:
You're setting the config object in the constructor in line 288, but not writing it to the Parcel here.

Reply:
yes, config is not being written on purpose.

Reply:
Updated to write config to parcel.